src/Libraries/CoreNodeModels/Input/DoubleSlider.cs

```diff
@@ -95,19 +94,25 @@ protected override void DeserializeCore(XmlElement nodeElement, SaveContext cont
 
         #endregion
 
+        private static double ConvertBack(string value)
+        {
+            double val = 0.0;
+            double.TryParse(value, NumberStyles.Any, CultureInfo.InvariantCulture, out val);
+            return val;
+        }
+
```

Comment:
DoubleDisplay is an IValueConverter, an interface derived from PresentationCore (yuck).  Here it is inlined.